nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/bin/nifi-env.sh

```diff
@@ -50,6 +50,9 @@ export NIFI_PID_DIR
 NIFI_LOG_DIR="$(setOrDefault "$NIFI_LOG_DIR" "$NIFI_HOME/logs")"
 export NIFI_LOG_DIR
 
+# Disable automatic Logback Initializer to avoid shutdown on web application termination
```

Comment:
It might be good to document alternatives to the use of the environment variable to activate this behavior.  I noticed this issue in the context of working on the system tests.  There is a `bootstrap.conf` available to set system properties, for example.

Reply:
Thanks for feedback @greyp9! The Logback documentation describes the configuration options under the [WebShutdownHook](https://logback.qos.ch/manual/configuration.html#webShutdownHook) section. Will update this pull request to include changes to bootstrap.conf in the system tests.